### PR TITLE
Week2 nadin

### PR DESCRIPTION
**Pull request by Nadoshka**

### CODE REVIEW

File: homework-submission/week2/App.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    const todoList = this.state.todoList.filter(item => {
+      return item.id !== parseInt(event.target.id);
+    });
+    this.setState({ todoList });
+  };
+
+  render() {
+    return (
+      <div className="panel">
+        namicList items={this.state.todoList} deleteItem={this.deleteItem} />
```
[the remark this answers is not in the document]
Comment (Nadoshka) [reply homework-submission/week2/App.js]: Indeed 🤦🏻‍♀️ fixed thanks for you comments

**Comment by fedefu** [comment]
@Nadoshka can you please implement feedback so we can close this PR? Thanks!

File: homework-submission/week2/App.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
+//the goal of this component is to create a list item
+class Item extends React.Component {
```
[the remark this answers is not in the document]
Comment (Nadoshka) [reply homework-submission/week2/App.js]: Thanks for your feedback! It's done! 👍

File: homework-submission/week2/App.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+            <Item description={item.description} deadline={item.deadline} done={item.done} />
+            <Button id={item.id} type="button" text="Delete" onClick={deleteItem} />
+          </div>
+        ))}
+      </ul>
+    );
+  }
+}
+
+const UserInput = ({ addItem }) => {
```
[the remark this answers is not in the document]
Comment (Nadoshka) [reply homework-submission/week2/App.js]: Thanks for the link! It's done! 👍

File: homework-submission/week2/App.js [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+      ],
+    };
+  }
+  addItem = event => {
+    event.preventDefault();
+    const description = event.target.description.value;
+    const deadline = event.target.date.value;
+    const todoList = [...this.state.todoList]; //copy the old state array to the new array
+    todoList.push({
+      id: todoList[todoList.length - 1].id + 1,
```
[the remark this answers is not in the document]
Comment (Nadoshka) [reply homework-submission/week2/App.js]: Good to know! Thanks 👍 :)

**Comment by ArcoMul** [comment]
Great work @Nadoshka!

**Pull request closed without merging**